core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -155,6 +171,27 @@ public synchronized ConfigCheckReport getConfigCheckReport() {
     return mConfigCheckReport;
   }
 
+  /**
+   * Logs the configuration check report information.
+   */
+  public synchronized void logConfigReport() {
+    Status reportStatus = mConfigCheckReport.getStatus();
+    if (reportStatus.equals(Status.PASSED)) {
+      LOG.info("Stauts: {}", reportStatus);
+    } else if (reportStatus.equals(Status.WARN)) {
+      LOG.warn("Status: {}. {}", reportStatus, HELP_INFO);
+      LOG.warn("Warnings: {}", mConfigCheckReport.getConfigWarns().values().stream()
+          .map(Object::toString).collect(Collectors.joining(", ")));
+    } else {
+      LOG.error("Status: {}. {}", reportStatus, HELP_INFO);
```

Comment:
The log message will look like "<timestamp> ERROR ServerConfigurationChecker - Status: ERROR". The second "Status: ERROR" is superfluous. Better to write something that adds more meaning like "Inconsistent configuration detected."

Reply:
Changed to 
```
  private static final String CONSISTENT_CONFIGURATION_INFO
      = "All server-side configuration are consistent.";
  private static final String INCONSISTENT_CONFIGURATION_INFO
      = "Inconsistent configuration detected. "
      + "Only a limited set of inconsistent configuration will be shown here. "
      + "For details, please visit Alluxio web UI or "
      + "run fsadmin doctor CLI.";
```